firefox-ios/Client/Frontend/Autofill/Address/AddressCellView.swift

```diff
@@ -9,6 +9,15 @@ import Storage
 
 import struct MozillaAppServices.Address
 
+extension VerticalAlignment {
```

Comment:
Even though its small, I would recommend moving it to our extensions folder - https://github.com/mozilla-mobile/firefox-ios/tree/main/firefox-ios/Client/Extensions